docs/architecture.rst

```diff
@@ -363,6 +363,374 @@ As a reference, below is an example of a Profile and the corresponding secret.
     example_key_id: <access key>
     example_secret_access_key: <access secret>
 
+RepositoryServers
+-----------------
+
+RepositoryServer CR is required by kanister controller to start
+a kopia repository server. The CR has list of parameters to configure
+kopia repository server.
```

Comment:
We are introducing the new term kopia with these docs. We should add a short para and references about Kopia and how we are using it. It would be helpful for a newcomer who doesn't have any idea about it.

Reply:
Yes I will be adding that at the start of architecture.rst